using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md

```diff
@@ -22,13 +22,13 @@ Here are the key-value pairs in this event:
 
 The `registerDevice` API parameters are also sent in the event, as described below. The `deviceinfo` map is internally populated.
 
-```text
+```json
 {
-    "registerdevice": <Boolean>
-    "devicetoken": <String>
-    "userkey": <String>
-    "additionalparameters": <VariantMap>
-    "deviceinfo": <StringMap>
+    "registerdevice": "<Boolean>",
+    "devicetoken": "<String>",
+    "userkey": "<String>",
+    "additionalparameters": "<VariantMap>",
+    "deviceinfo": "<StringMap>"
```

Comment:
Adding quotes in a json example may indicate that the value is a string for a developer. Can we revert this change in all similar examples?

Reply:
Hmm, I know speaking as a person who does do development that I don't think I'd be confused by that?

Let me check what we do on the Platform-side of things, and get back to you on that.